dotnet/src/Connectors/Connectors.Amazon/Bedrock/Settings/AmazonClaudeExecutionSettings.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.SemanticKernel.Text;
+
+namespace Microsoft.SemanticKernel.Connectors.Amazon;
+
+/// <summary>
+/// Prompt execution settings for Anthropic Claude Text Generation
+/// </summary>
+[JsonNumberHandling(JsonNumberHandling.AllowReadingFromString)]
+public class AmazonClaudeExecutionSettings : PromptExecutionSettings
+{
+    private int _maxTokensToSample;
+    private List<string>? _stopSequences;
+    private float? _temperature;
+    private float? _topP;
+    private int? _topK;
+
+    /// <summary>
+    /// Default max tokens for a text generation.
+    /// </summary>
+    public static int DefaultTextMaxTokens { get; } = 200;
```

Comment:
Should this be static? It's also not possible to change it outside, so I think it's better to define it as `private const`.